### PR TITLE
RN-1.2.0

### PR DESCRIPTION
**Pull request by andreipope**
RN-1.2.0

### CODE REVIEW

File: content/release-notes.md [hunk 1]
```diff
@@ -7,6 +7,42 @@ hidesections: true
...
+| PB-958 | On the **Applications** page, the list of resource types is now sorted alphabetically. |
+| PB-966 | On the **Schedules** page, PX-Backup now displays the namespaces included in a backup. |
+
+### Fixes
+
+Portworx, Inc. has fixed the following issues:
+
+|**Issue Number**|**Issue Description**|
+|----|----|
+| PB-659 | PX-Backup did not notify the users when their license was about to expire. </br></br> **User Impact:** The backups were failing. </br></br> **Resolution:** PX-Backup now displays a warning seven days before license expiration. |
```
Comment (bhavana-ar) [inline content/release-notes.md:42]: Impact: The user would know about expired license only when backups and restores started failing due to expired license.

File: content/release-notes.md [hunk 2]
```diff
@@ -7,6 +7,42 @@ hidesections: true
...
+Nov 30, 2020
+### New features
+
+- Introducing **usage-based pricing** for remote cluster nodes.
+- PX-Backup now supports generic CSI driver backup and restore.
+- Introducing cluster-level aggregated metrics for backup and restore with Prometheus metrics and Grafana dashboards
+
+### Improvements
+
+Portworx, Inc. has upgraded or enhanced functionality in the following areas:
```
Comment (bhavana-ar) [inline content/release-notes.md:21]: We should not be saying portworx, Inc anymore.

File: content/release-notes.md [hunk 3]
```diff
@@ -7,6 +7,42 @@ hidesections: true
...
+| PB-948 | PX-Backup now displays the date and the time when the next backup will run in the modal containing details about your backup schedule. |
+| PB-946 | PX-Backup now uses the same format to display dates for backups, restores, and backup schedules. |
+| PB-947 | On the **Scheduled Backup Details** pane, the **NEXT SCHEDULED BACKUP ON** field now shows the date and the time when the next backup will run. When a backup is running, the **NEXT SCHEDULED BACKUP ON** field shows **In Progress**. |
+| PB-950 | If you select the name of a backup, restore, or backup schedule from the table view, PX-Backup now displays a modal containing details about your backup, restore, or backup schedule. |
+| PB-958 | On the **Applications** page, the list of resource types is now sorted alphabetically. |
+| PB-966 | On the **Schedules** page, PX-Backup now displays the namespaces included in a backup. |
+
+### Fixes
+
+Portworx, Inc. has fixed the following issues:
```
Comment (bhavana-ar) [inline content/release-notes.md:38]: Again, remove Portworx, Inc here

File: content/release-notes.md (hunk 1 above)
Comment (andreipope) [reply content/release-notes.md]: @bhavana-ar It's updated.

File: content/release-notes.md (hunk 3 above)
Comment (andreipope) [reply content/release-notes.md]: @bhavana-ar I think you're right. Anyway, let me check with Nathan to see what's his take. Once I get an answer, I'll update this PR.

**Pull request merged**